src/window_popup.js

```diff
@@ -0,0 +1,54 @@
+export default function DialogWindow() {
+    let d = new Dialog({
+        title: "Module Aktiviert !",
+        content: `
+        <div style="text-align: justify;">
+            <h2>
+                Willkommen in der deutschen Übersetzung des DnD5e Systems
+            </h2>
+            <p>
+                Wir freuen uns, dass du unser Modul verwendest, um dein Spielsystem ins Deutsche zu übersetzen. 
+            </p>
+            <p>
+                Wir arbeiten gerade an der Fertigstellung, der restlichen Texte. Das bedeutet, dass noch nicht alle Texte auf Deutsch verfügbar sind. 
+                Falls du einen Fehler findest, kannst du diesen <a href="https://github.com/League-of-Foundry-Developers/foundryvtt-dnd5e-lang-de/issues" target="_blank">hier melden</a>.
+            </p>
+            <p>
+                Wenn du uns helfen oder unterstützen möchtest, kannst du gerne auf unsere <a href="https://translator.gilneas.at/" target="_blank">Übersetzungsseite</a> kommen und mithelfen.
+            </p>
+            <p>
+                Jede Hilfe ist eine große Unterstützung für uns und für die Community.
+            </p>
+            <p>
+                Das Übersetzungstool befindet sich aktuell in einer offenen Beta. Wenn du fehler findest, kannst du ihn <a href="https://github.com/League-of-Foundry-Developers/foundryvtt-dnd5e-lang-de-ui/issues" target="_blank">hier einmelden.</a> 
+            </p>
+            <hr> 
+            <h3>
+                Wir bedanken uns bei:
+            </h3>
+            <p>
+                <i>Fallayn, Crash, AlexElbracht, JPrince, Sinon, Zeku, vttom</i>
```

Comment:
```suggestion
                <i>Fallayn, Crash, AlexElbracht, JPrince, Sinon, Zeku, vttom, Hydroxi, Smicman, ThoGri, Morvar</i>
```
Würde hier ruhig Hydroxi, Smicman, ThoGri und Morvar noch mit nennen, die wir auch in der [module.json](https://github.com/League-of-Foundry-Developers/foundryvtt-dnd5e-lang-de/blob/master/module.json) erwähnen, auch wenn sie zur Zeit nicht mehr aktiv beitragen.